Makefile

```diff
@@ -109,3 +116,35 @@ e2e: e2e-ipv4-iptables \
 	e2e-ipv4-nft \
 	e2e-ipv6-nft \
 	e2e-dual-nft
+
+## Build binary for Windows platform
+windows: PLATFORM="windows"
+windows:
```

Comment:
we're not actually building or testing on windows - this wont be usable

Reply:
The idea was have a make target for windows to build kpng to test the new backend `userspace` + `winkernel` on windows.